app/assets/stylesheets/home.css.scss

```diff
@@ -1,24 +1,59 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+// TO-DO: Media queries: Make generate q button left for desktop, center for mobile
+// Make cover photo height longer for desktop
+
 .jumbotron {
   background-image: image-url('orange.jpg');
-  background-size:cover;
-  height: 52vw;
+  background-size: cover;
   color: white;
   //line-height: -35px;
+//  @media (min-width :@screen-md-min){
+//    height: 20em;
+//  }
+
+
+  .container-row{
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 0;
+    margin-bottom: 0;
+    max-width: 62.5rem;
```

Comment:
Using `rem` here, but for font and margin settings (below) using `em`. Why the difference?